pkg/oci/fix_owner_linux.go

```diff
@@ -21,42 +21,27 @@ import (
 	"syscall"
 )
 
-func fixOwner(targetFile, modelFile string) error {
-	info, err := os.Stat(modelFile)
+func fixOwner(targetFiles ...string) error {
+	cwd, err := os.Getwd()
```

Comment:
This relies on the chdir done previously. This is fine but it is done in a different package. Could the function be named `fixOwnerWithCwd` to document this requirement?

Reply:
I changed the approach not to rely on this. Now the function receives a "model file" which is used to change the owner on all other files. It's a callers responsibility to pass this file.